[rospy.client][INFO] 2017-06-10 22:58:11,578: init_node, name[/rosbridge_websocket], pid[12954]
[xmlrpc][INFO] 2017-06-10 22:58:11,578: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2017-06-10 22:58:11,590: Started XML-RPC server [http://ip-172-31-19-100:58156/]
[rospy.impl.masterslave][INFO] 2017-06-10 22:58:11,593: _ready: http://ip-172-31-19-100:58156/
[xmlrpc][INFO] 2017-06-10 22:58:11,597: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2017-06-10 22:58:11,591: ROS Slave URI: [http://ip-172-31-19-100:58156/]
[rospy.registration][INFO] 2017-06-10 22:58:11,597: Registering with master node http://localhost:52840
[rospy.init][INFO] 2017-06-10 22:58:11,706: registered with master
[rospy.rosout][INFO] 2017-06-10 22:58:11,710: initializing /rosout core topic
[rospy.rosout][INFO] 2017-06-10 22:58:11,720: connected to core topic /rosout
[rospy.simtime][INFO] 2017-06-10 22:58:11,725: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2017-06-10 22:58:11,751: Rosbridge WebSocket server started on port 60305
[rospy.internal][INFO] 2017-06-10 22:58:11,965: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2017-06-10 22:58:18,424: Client connected.  1 clients total.
[rosout][INFO] 2017-06-10 22:58:18,572: [Client 0] Subscribed to /tf2_web_republisher/feedback
[rospy.internal][INFO] 2017-06-10 22:58:18,581: topic[/tf2_web_republisher/feedback] adding connection to [http://ip-172-31-19-100:35504/], count 0
[rosout][INFO] 2017-06-10 22:58:18,599: [Client 0] Subscribed to /visualization_marker_array
[rospy.internal][INFO] 2017-06-10 22:58:18,602: topic[/visualization_marker_array] adding connection to [http://ip-172-31-19-100:58210/], count 0
[rosout][INFO] 2017-06-10 22:58:18,618: [Client 0] Subscribed to /control_markers/tunneled/update
[rospy.internal][INFO] 2017-06-10 22:58:18,635: topic[/control_markers/tunneled/update] adding connection to [http://ip-172-31-19-100:43084/], count 0
[rospy.internal][INFO] 2017-06-10 22:58:18,714: topic[/control_markers/feedback] adding connection to [/marker_control_13197_1497160694578], count 0
[rospy.internal][INFO] 2017-06-10 22:58:18,723: topic[/tf2_web_republisher/goal] adding connection to [/tf2_web_republisher], count 0
[rosout][INFO] 2017-06-10 22:58:18,731: [Client 0] Subscribed to /tf2_web_republisher/feedback
[rosout][INFO] 2017-06-10 22:58:18,736: [Client 0] Subscribed to /tf2_web_republisher/feedback
[rosout][INFO] 2017-06-10 22:58:18,741: [Client 0] Subscribed to /tf2_web_republisher/feedback
[rosout][INFO] 2017-06-10 22:58:18,745: [Client 0] Subscribed to /tf2_web_republisher/feedback
[rosout][INFO] 2017-06-10 22:58:18,749: [Client 0] Subscribed to /tf2_web_republisher/feedback
[rosout][INFO] 2017-06-10 22:58:18,754: [Client 0] Subscribed to /tf2_web_republisher/feedback
[rosout][INFO] 2017-06-10 22:58:18,758: [Client 0] Subscribed to /tf2_web_republisher/feedback
[rospy.internal][INFO] 2017-06-10 22:58:18,823: topic[/tf2_web_republisher/cancel] adding connection to [/tf2_web_republisher], count 0
[rospy.internal][WARNING] 2017-06-10 23:03:12,255: Unknown error initiating TCP/IP socket to ip-172-31-19-100:46318 (http://ip-172-31-19-100:35504/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 555, in connect
    self.socket.connect((dest_addr, dest_port))
  File "/usr/lib/python2.7/socket.py", line 224, in meth
    return getattr(self._sock,name)(*args)
error: [Errno 111] Connection refused

[rospy.internal][INFO] 2017-06-10 23:03:12,255: topic[/tf2_web_republisher/feedback] removing connection to http://ip-172-31-19-100:35504/
[rospy.internal][WARNING] 2017-06-10 23:03:12,255: Unknown error initiating TCP/IP socket to ip-172-31-19-100:52408 (http://ip-172-31-19-100:43084/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 555, in connect
    self.socket.connect((dest_addr, dest_port))
  File "/usr/lib/python2.7/socket.py", line 224, in meth
    return getattr(self._sock,name)(*args)
error: [Errno 111] Connection refused

[rospy.internal][WARNING] 2017-06-10 23:03:12,260: Unknown error initiating TCP/IP socket to ip-172-31-19-100:43901 (http://ip-172-31-19-100:58210/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 555, in connect
    self.socket.connect((dest_addr, dest_port))
  File "/usr/lib/python2.7/socket.py", line 224, in meth
    return getattr(self._sock,name)(*args)
error: [Errno 111] Connection refused

[rospy.internal][INFO] 2017-06-10 23:03:12,262: topic[/control_markers/tunneled/update] removing connection to http://ip-172-31-19-100:43084/
[rospy.internal][INFO] 2017-06-10 23:03:12,266: topic[/visualization_marker_array] removing connection to http://ip-172-31-19-100:58210/
[rospy.core][INFO] 2017-06-10 23:03:12,332: signal_shutdown [signal-2]
[rospy.internal][INFO] 2017-06-10 23:03:12,435: topic[/tf2_web_republisher/cancel] removing connection to /tf2_web_republisher
[rospy.internal][INFO] 2017-06-10 23:03:12,439: topic[/tf2_web_republisher/goal] removing connection to /tf2_web_republisher
[rospy.internal][INFO] 2017-06-10 23:03:12,442: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2017-06-10 23:03:12,446: topic[/control_markers/feedback] removing connection to /marker_control_13197_1497160694578
[rospy.impl.masterslave][INFO] 2017-06-10 23:03:12,450: signal-2
[rospy.core][INFO] 2017-06-10 23:03:12,969: signal_shutdown [atexit]
